.github/workflows/check-build-test.yml

```diff
@@ -34,6 +42,7 @@ jobs:
       - run: util/style/test_indent_comments.sh
```

Comment:
Wouldn't you want to use `test_all.sh` here now?

Reply:
Done, and expanded with automated patch generation